A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -264,5 +276,29 @@ class CDUInitPage {
             CDUInitPage.ShowPage1(mcdu);
         };
     }
+
+    // Defining as static here to avoid duplicate code in CDUIRSInit
+    static ConvertDDToDMS(deg, lng) {
+            // converts decimal degrees to degrees minutes seconds
+            const M=0|(deg%1)*60e7;
+            let degree;
+            if (lng) {
+                degree = this.pad(0 | (deg < 0 ? deg = -deg:deg), 3, 0)
+            } else {
+                degree = 0 | (deg < 0 ? deg = -deg:deg);
+            }
+            return {
+                dir : deg<0 ? lng ? 'W':'S' : lng ? 'E':'N',
+                deg : degree,
+                min : Math.abs(0|M/1e7),
+                sec : Math.abs((0|M/1e6%1*6e4)/100)
+            };
+    }
+
+    static pad(n, width, filler) {
```

Comment:
If I understand correctly, this is the same functionality as the standard function String.prototype.padStart(), used in a lot of code already.

Reply:
Correct, modified accordingly